library/src/main/java/com/google/android/exoplayer/text/CuePainter.java

```diff
@@ -350,4 +362,164 @@ private static boolean areCharSequencesEqual(CharSequence first, CharSequence se
     return first == second || (first != null && first.equals(second));
   }
 
+  /**
+   * Gets position anchor given an anchor and an alignment value. This method takes into
+   * account that the parameter position anchor could be not set and then anchor "inherits" its
```

Comment:
The inheritance behavior is a property of WebVTT and so should be implemented in WebvttParser rather than here. It's actually implemented there already (see the alignmentToAnchor method).